client/client.go

```diff
@@ -33,14 +33,14 @@ const (
 )
 
 func init() {
-	data.SetDefaultExpiryTimes(notary.NotaryDefaultExpiries)
+	data.SetDefaultExpiryTimes(data.NotaryDefaultExpiries)
```

Comment:
Ah my apologies. I was also confusing `NotaryDefaultExpiries` with `defaultExpiryTimes` (the var that was already in the data package). This is fine for now, we have some tech debt we need to pay off that will require reassessing where some of our types are defined. It goes beyond the scope of this PR so this will work for now.